vue/assets/icons/font_trimming.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7 7.786V6h2.056v.857H7.703v.929H7zm0 2.857v2.714h.667v-2.714H7zm0 5.571V18h2.056v-.857h-1.39v-.929H7zM11.278 6h2.11v.857h-2.11V6zm4.333 0v.857h.722v.929H17V6h-1.389zm-4.333 11.143h2.11V18h-2.11v-.857zm5.055 0h-.722V18H17v-1.786h-.667v.929zm0-6.5v2.714H17v-2.714h-.667z" fill="#57626E"/><path fill-rule="evenodd" clip-rule="evenodd" d="M15.571 16h-.798a.332.332 0 01-.224-.073.446.446 0 01-.128-.184l-.713-1.931h-3.422l-.713 1.931a.392.392 0 01-.123.179.33.33 0 01-.223.078h-.799l3.05-8h1.043l3.05 8zm-2.15-2.97l-1.198-3.255A6.89 6.89 0 0112 9.027a6.176 6.176 0 01-.23.754l-1.197 3.249h2.848z" fill="#57626E"/></svg>
```

Comment:
Move these under `extras`. The root is reserved for our "base" icon set (Ikonate)